assets/html/evolution/tooltip_scale_from_corner.html

```diff
@@ -0,0 +1,25 @@
+<!doctype html>
+<html class="no-js" lang="en-us">
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="x-ua-compatible" content="ie=edge">
+  <meta name="author" content="{{YOUR NAME HERE}}">
+  <title>{{NAME OF YOUR COMPONENT}}</title>
+  <meta name="description" content="">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Roboto+Slab|Roboto:300,400,700" rel="stylesheet">
+  <link href="../../styles/main.css" rel="stylesheet">
+</head>
+<body>
+<button class="btn btn-primary tt-scale">Projects
+  <span class="tt-scale__vertical"><div class="tt-scale__content">Lorem ipsum, consectetur adipiscing elit. Donec metus orci. faucibus sem nec. Quisque tempus pretium dolor vitae tincidunt.</div>
```

Comment:
@hiepvo 
Placing `div` inside a `span` is invalid markup. Please change that. Span is an inline element and can hold only other inline elements, div is a block level element and it is not valid to place it inside an inline element regardless if you set its display property to inline with css.